config/locales/idv/en.yml

```diff
@@ -216,6 +217,17 @@ en:
         learn_more_verify_in_person: Learn more about verifying in person
         supported_documents: See a list of accepted state-issued IDs
         verify_by_mail: Verify your address by mail instead
+    unavailable:
+      exit_button: 'Exit %{app_name}'
+      idv_explanation:
+        with_sp: '%{sp} needs to make sure you are you — not someone pretending to be
+          you.'
+        without_sp: 'The agency that you are trying to access needs to make sure you are
+          you — not someone pretending to be you.'
```

Comment:
Can we reuse [this existing translation](https://github.com/18F/identity-idp/blob/19c0af192faea575a6549cc15e394cc0a3d98032/config/locales/doc_auth/en.yml#L195-L196) here? We could add "the agency you are trying to access" as a separate phrase. It exists as part of other translations.

Reply:
I'm inclined not to get too DRY with this translation stuff. I'd like to keep the translations for a single screen grouped together rather than include doc auth translations on this page.